components/Snackbar/src/MDCSnackbarMessageView.h

```diff
@@ -101,3 +107,6 @@ __deprecated_msg("Use messsageTextColor instead.");
 
 @end
 // clang-format on
+
+@interface MDCSnackbarMessageViewButton : MDCFlatButton
```

Comment:
Can we keep this as a private implementation detail? I see you're exposing MDCButton instances above so am wondering if we can continue hiding the subclass implementation.

Reply:
done. the idea behind this is if snackbar has specific properties for its button in the future, it won't be exposed and we won't be able to theme those properties.